_studio/mfx_lib/shared/src/mfx_mpeg2_encode_vaapi.cpp

```diff
@@ -193,7 +193,9 @@ namespace
 
         sps.intra_period    = pExecuteBuffers->m_GOPPictureSize; // 22??
         sps.ip_period       = pExecuteBuffers->m_GOPRefDist;
-        sps.bits_per_second = winSps.bit_rate; // 104857200;
+        // For VBR maxBitrate should be used as sps parameter, target bitrate has no place in sps, only in BRC
+        sps.bits_per_second = (winSps.RateControlMethod == MFX_RATECONTROL_VBR && winSps.MaxBitRate > winSps.bit_rate) ?
+                winSps.MaxBitRate : winSps.bit_rate; // 104857200;
```

Comment:
104857200 - what is this magic number? please, either remove or add a clear comment.